src/controllers/HeuristicController.js

```diff
@@ -1,101 +1,118 @@
-/* 
-import {HeuristicController} from '././HeuristicController'
-
-let HeuristicController = new HeuristicController()
-*/
-
 // imports
 
-import api from "@/api/index";
-import database from "../api/modules/database";
-
-import Heuristic from "../models/Heuristic";
-import HeuristicAnswer from "../models/HeuristicAnswer";
-import HeuristicQuestion from "../models/HeuristicQuestion";
-import HeuristicQuestionAnswer from "../models/HeuristicQuestionAnswer";
-import HeuristicQuestionDescription from "../models/HeuristicQuestionDescription";
-import HeuristicTest from "../models/HeuristicTest";
+import Heuristic from "@/models/Heuristic";
+import HeuristicAnswer from "@/models/HeuristicAnswer";
+import HeuristicQuestion from "@/models/HeuristicQuestion";
+import HeuristicQuestionAnswer from "@/models/HeuristicQuestionAnswer";
+import HeuristicQuestionDescription from "@/models/HeuristicQuestionDescription";
 
+// functions
 
+function toHeuristic(){
```

Comment:
Why are these functions outside the controller class? Is there an specific reason or it is because you had trouble calling it from another file?